.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -30,6 +30,10 @@ about: Report broken or incorrect behaviour
 
 ### System Information
 
-- discord.py version:
-- Python version:
-- Operating system:
+<!-- Do `python -m discord -v` and paste this information.-->
```

Comment:
I'm not sure if "do" is the most appropriate here. It might not be obvious to beginners that this is a shell command.

Reply:
I don't think `at` is what should be used for that; in or on would be better.